src/gateway/sharding/mod.rs

```diff
@@ -62,34 +62,28 @@ use crate::model::gateway::{GatewayIntents, ShardInfo};
 use crate::model::id::{ApplicationId, GuildId, ShardId};
 use crate::model::user::OnlineStatus;
 
-/// A Shard is a higher-level handler for a websocket connection to Discord's gateway.
+/// An abstract handler for a websocket connection to Discord's gateway.
 ///
-/// The shard allows for sending and receiving messages over the websocket,
-/// such as setting the active activity, reconnecting, syncing guilds, and more.
+/// Allows a user to send and receive messages over said websocket, such as: setting the current
+/// activity, reconnecting, syncing guilds, and more.
```

Comment:
Could this be expanded to bullet points with links to the methods?

Reply:
Done.